src/main/java/racingcar/Application.java

```diff
@@ -1,7 +1,90 @@
 package racingcar;
 
+import camp.nextstep.edu.missionutils.Console;
+import camp.nextstep.edu.missionutils.Randoms;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
 public class Application {
     public static void main(String[] args) {
         // TODO: 프로그램 구현
+
+        System.out.println("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)");
+        String carNames = Console.readLine();
+        System.out.println("시도할 회수는 몇회인가요?");
+        //사용자는 몇 번의 이동을 할 것인지를 입력할 수 있어야 한다.
```

Comment:
주석 달아서 설명해주신 부분 좋습니다!
추가적으로 코드의 일관성을 위해서 작성하시게 된다면 규칙을 정하여 포맷팅을 진행해보세요!
```suggestion
        // 사용자는 몇 번의 이동을 할 것인지를 입력할 수 있어야 한다.
```